modules/administration/pages/ssl-certs-hsts.adoc

```diff
@@ -3,21 +3,33 @@
 
 HTTP Strict Transport Security (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security[HSTS]) is a policy mechanism that helps to protect websites against man-in-the-middle attacks such as protocol downgrade attacks and cookie hijacking.
 
+
 {productname} allows enabling HSTS, to enable it for a {productname} Server:
 
 .Procedure
-1. Edit `/etc/apache2/conf.d/zz-spacewalk-www.conf`
-2. Uncomment the line `# Header always set Strict-Transport-Security "max-age=63072000; includeSubDomains"`
+1. Create new configuration file in  `/etc/apache2/conf.d/<filename>.conf`, for example `/etc/apache2/conf.d/zzz-spacewalk-www.conf` or `/etc/apache2/conf.d/zz-spacewalk-www-custom.conf`.
+2. Add line `Header always set Strict-Transport-Security "max-age=63072000; includeSubDomains"`
 3. Restart Apache with `systemctl restart apache2`
 
+
+
 To enable it for {productname} Proxies:
 
 .Procedure
-1. Edit `/etc/apache2/conf.d/spacewalk-proxy.conf`
-2. Uncomment the line `# Header always set Strict-Transport-Security "max-age=63072000; includeSubDomains"`
+1. Create new configuration file in  `/etc/apache2/conf.d/<filename>.conf`, for example `/etc/apache2/conf.d/zzz-spacewalk-proxy.conf` or `/etc/apache2/conf.dz/zz-spacewalk-proxy-custom.conf`.
```

Comment:
`/etc/apache2/conf.d/zzz-spacewalk-proxy.conf` is not a new file. We ship this file and it must not be changed or overwritten.